k8s_handle/k8s/diff.py

```diff
@@ -0,0 +1,83 @@
+import sys
+import logging
+from difflib import ndiff
+from datetime import datetime
+from functools import reduce
+import operator
+import yaml
+from .adapters import Adapter
+from k8s_handle.templating import get_template_contexts
+log = logging.getLogger(__name__)
+
+IGNORE_FIELDS = [
+    'metadata.annotations:kubectl.kubernetes.io/last-applied-configuration',
+    'metadata.annotations:deployment.kubernetes.io/revision',
+    'metadata:creationTimestamp',
+    'metadata:resourceVersion',
+    'metadata:selfLink',
+    'metadata:uid',
+    'metadata:namespace',
```

Comment:
Very often, this value can be overridden. Can leave it to compare?

Reply:
Unfortunatly we don't known the namespace at the diff phase due to we apply namespace in api calls(patch_namespaced_<object>)